### PR TITLE
Rate mars and uranus

### PR DESCRIPTION
**Pull request by steven-murray**
I rated them, please @akvydula let me know what you think.

### CODE REVIEW

File: mars.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ Red planet
 The two moons WILL be a problem
 
 New Idea: mars sucks
+
+RATING: 1
```
Comment (steven-murray) [inline mars.txt]: I really disagree. Please give it a -3.

File: uranus.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 stuff
+
+RATING: 2
```
Comment (steven-murray) [inline uranus.txt]: Make this a 10!!

**Comment by akvydula** [comment]
How about a scale of 100?

**Review by steven-murray — COMMENTED** [review]
Also, add rating for neptune!

**Review by akvydula — APPROVED** [review]

**Pull request merged**